templates/freeradius/openwisp_site.j2

```diff
@@ -0,0 +1,69 @@
+server default {
+    listen {
+        type = auth
+        ipaddr = *
+        port = 0
+        limit {
+            max_connections = 16
+            lifetime = 0
+            idle_timeout = 30
+        }
+    }
+
+    listen {
+        ipaddr = *
+        port = 0
+        type = acct
+        limit {}
+    }
+
+    authorize {
+        rest
+        sql
+        dailycounter
+        noresetcounter
+        dailybandwidthcounter
+    }
+
+    authenticate {
+        Auth-Type PAP {
+                pap
+        }
+
+        Auth-Type CHAP {
+                chap
+        }
+
+        Auth-Type MS-CHAP {
+                mschap
+        }
+
+        Auth-Type EAP {
+                eap
+        }
+    }
```

Comment:
After reading: https://community.privacyidea.org/t/fresh-install-pid-but-i-get-auth-type-eap-section/1067/5

I searched for activating EAP Authentication module works, I'll look it but up but please check it out, my knowledge of freeradius is very limited and even small changes seem to break it completely.